# HG changeset patch
# User Rafael Laboissiere <rafael@debian.org>
# Date 1243977356 -7200
#      Tue Jun 02 23:15:56 2009 +0200
# Node ID eaf4e71e90e8d5be01f00993c04c5917af9a2fff
# Parent  7621f0b0e588f0e7b206cbaca5c6452db9798190
In test, look also for the gzipped version of the info_file

The info program accepts .info files that are compressed with gzip. Consider
also the info_file with the suffix ".gz" when doing the test.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2009-06-02  Rafael Laboissiere  <rafael@debian.org>
+
+	* help/doc.m: In test, look also for the gzipped version of the
+	info_file
+
 2009-05-29  John W. Eaton  <jwe@octave.org>
 
 	* plot/__gnuplot_get_var__.m: Insert missing semicolon.
 
 2009-05-28  Ben Abbott <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Set x2range when xaxislocation=='top' and set
 	y2range when yaxislocation=='right'. Simplified support for ticklabel
diff --git a/scripts/help/doc.m b/scripts/help/doc.m
--- a/scripts/help/doc.m
+++ b/scripts/help/doc.m
@@ -101,11 +101,11 @@ function retval = doc (fname)
     endif
 
   else
     print_usage ();
   endif
 
 endfunction
 
-%!test if exist( info_file ()) != 2
-%!       error ("Info file %s does not exist!", info_file ());
+%!test if exist( info_file ()) != 2 && exist (sprintf ("%s.gz", info_file ())) != 2
+%!       error ("Info file %s or %s.gz does not exist!", info_file (), info_file ());
 %!     endif
